packer/builder/azure/driver_restapi/request/RetryPolicy.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package request
+
+import (
+	"log"
+	"strings"
+	"time"
+
+	"github.com/MSOpenTech/packer-azure/packer/builder/azure/driver_restapi/response/model"
+)
+
+func defaultRetryPolicy() retryPolicy {
+	return retryPolicy{
+		exponentialBackoff("Throttling", func(err model.AzureError) bool {
+			return err.Code == "TooManyRequests"
+		}, 5*time.Second, 2*time.Minute, 0),
+		constantBackoff("MissingOrIncorrectVersionHeader", func(err model.AzureError) bool {
+			return err.Code == "MissingOrIncorrectVersionHeader" // shouldn't occur, since we always add it
+		}, 0, 100),
```

Comment:
yep... an erroneous error... I don't think I've ever seen it. I would like to remove it here and file bugs for RDFE if anyone runs into it?